compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1325,6 +1325,17 @@ pub enum CastKind {
     IntToFloat,
     PtrToPtr,
     FnPtrToPtr,
+    /// Convert a raw pointer to an `impl Pointee<Metadata = M>` into `M`.
```

Comment:
"Convert raw pointer ... to" is quite confusing here.
```suggestion
    /// Convert a `*const/mut impl Pointee<Metadata = M>` into `M`.
```